config/ldp/modes/default.json

```diff
@@ -7,7 +7,24 @@
       "@type": "WaterfallHandler",
       "handlers": [
         { "@type": "MethodModesExtractor" },
-        { "@type": "SparqlPatchModesExtractor" }
+        {
+          "@type": "WrappedMethodModesExtractor",
```

Comment:
Hmm that took some parsing. Should we really do wrapping here?

I wonder if we can come to a solution with a `MethodFilter` or so, which basically is a generic handler that returns `canHandle` true only for listed methods.

I initially thought a `SequenceHandler` with a `MethodFilter` followed by `SparqlUpdateModesExtractor`, but then I saw that `The `canHandle` check of this handler will always succeed` (why though? we could at least check the first step?) so fair enough. But still, suggestion below on `WrappedMethodModesExtractor`.

Reply:
From what I recall, `SequenceHandler` and `ParallelHandler` were specifically made for situations where we had a bunch of handlers that always had to be executed, with `SequenceHandler` being used if the order mattered (e.g., middleware first).

I do agree that this change does add a bit of verbosity to the config, but perhaps this is just a result of how it is written here? Maybe it would be clearer if the `WrappedMethodModesExtractor` (or its generic alternative) would be defined in a separate block? In that case the main ModesExtractor would be a WaterfallHandler with 3 simple entries.

Reply:
check